gef-ui/server/controllers/application-details/index.js

```diff
@@ -249,9 +249,10 @@ const applicationDetails = async (req, res, next) => {
       params.errors = req.errors;
     }
 
-    if (req.success) {
-      params.success = req.success;
-    }
+    // Retrieve success message from req.flash
+    const [successMessage] = req.flash('success');
+
+    params.success = successMessage; // Assign the successMessage to params.success
```

Comment:
`req.flash('success') ` will not always have value. Can we wrap don't set params.success if `successMessage` is empty

Reply:
Added condition to check `successMessage` is empty or not.